pages/settings/apikeys.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <settings-page page-title="settings.apiKeys.apiKeys">
+    <template #actions>
+      <v-btn color="primary" @click="() => (addingNewKey = true)">
+        {{ $t('settings.apiKeys.addNewKey') }}
+      </v-btn>
+      <v-btn v-if="apiKeys.length" color="error" @click="revokeAllApiKeys">
+        {{ $t('settings.apiKeys.revokeAll') }}
+      </v-btn>
+    </template>
+    <template #content>
+      <v-col>
+        <v-data-table :headers="headers" :items="apiKeys" class="elevation-1">
+          <!-- eslint-disable-next-line vue/valid-v-slot -->
+          <template #item.DateCreated="{ item }">
+            <span>
```

Comment:
```suggestion
            <span class="text-capitalize-first-letter">
```

Note: You'll need to move the class to one of the global stylesheets. It's already used in UserCard and logsAndActivity.